### PR TITLE
feat(plugin): added switch accounts component

### PR DESCRIPTION
**Pull request by TarasPotsikaylo**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

switch-account

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      new TooltipProperties('white', account.accountIndex, 50, 50, 'Copied!', 'black')
+    )
+  }
+
+  const shortingAddress = (address: string | number): string => {
+    const addressString: string = address.toString()
+    return `${addressString.slice(0, 4)}...${addressString.slice(-4)}`
+  }
+
+  const displayAccountIcon = () => {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx:136]: there is no need for this function they all return the same thing

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    navigator.clipboard.writeText(address.toString())
+    setCopiedWalletAddress(address)
+    setTooltipProperties(
+      new TooltipProperties('white', account.accountIndex, 50, 50, 'Copied!', 'black')
+    )
+  }
+
+  const shortingAddress = (address: string | number): string => {
+    const addressString: string = address.toString()
+    return `${addressString.slice(0, 4)}...${addressString.slice(-4)}`
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx:133]: Use `CryptoAddress` component

**Review by julianromero-bc — COMMENTED** [review]
It would be great to have a visual demo such as a gif or an image

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  border-radius: 50%;
+  border: 2px solid ${(props) => props.theme.grey800};
+`
+const WalletBlock = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  cursor: copy;
+`
+class TooltipProperties {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx:63]: Are you adding a new generic component?

I'd rather not add `classes`, as nowhere else in the app do we use this standard. Just use a function instead?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+      })
+    }
+
+    return addressR.cata({
+      Failure: () => null,
+      Loading: () => null,
+      NotAsked: () => null,
+      Success: (val) => val.address
+    })
+  }
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx:153]: For generating next address, why is STX different, just make them all the same

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  color: ${(props) => props.theme.grey400};
+`
+const SwitchAccount = (props) => {
+  const [selectedAccountIndex, setSelectedAccountIndex] = useState<number>(0)
+  const [copiedAccountIndex, setCopiedAccountIndex] = useState<string | number>('')
+  const { accounts, data } = props
+
+  const switchAccounts = [
+    accounts.ETH[0],
+    accounts.BTC[0],
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx:46]: why only the first account?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    accounts.BCH[0],
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx:47]: why only the first account?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+  setSelectedAccountIndex,
+  ...props
+}) => {
+  const state = useSelector((state: RootState) => state)
+  const balance = getBalanceSelector(account.coin)(state).getOrElse(0)
+  const [tooltipProperties, setTooltipProperties] =
+    useState<TooltipProperties>(defaultTooltipProperties)
+
+  useEffect(() => {
+    props.requestActions.getNextAddress(account)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx:114]: nice

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+    NotAsked: () => <Loading />,
+    Success: (value) => (
+      <BalanceText size='14px' weight={500}>
+        {`Total Balance ${value.totalBalance}`}
+      </BalanceText>
+    )
+  })
+
+  const closeSwitchAccountModal = () => {
+    // TODO: Close switch account modal window.
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/index.tsx:64]: finalize todo?

**Comment by schnogz** [comment]
please resolve the merge conflicts to base branch @TarasPotsikaylo

**Pull request closed without merging**